data/presets/building/waste_shed.json

```diff
@@ -0,0 +1,27 @@
+{
+    "icon": "maki-recycling",
+    "fields": [
+        "{building}",
+        "lockable",
+        "access"
+    ],
+    "geometry": [
+        "area"
+    ],
+    "locationSet": {
+        "include": ["DE", "EE", "SE"]
+    },
+    "tags": {
+        "building": "shed",
+        "amenity": "recycling",
```

Comment:
typically also non-recyled trash containers are there

Reply:
Would `waste_disposal` be better then and cover both?